src/Functions/Public/iaas/New-vRAProject.ps1

```diff
@@ -30,6 +30,12 @@
     .PARAMETER SharedResources
     Deployments are shared between all users in the project
 
+    .PARAMETER PlacementPolicy
+    Placement Policy
```

Comment:
Just a suggestion to add (e.g. DEFAULT or SPREAD) as I see it in the validation set but maybe for documentation purposes note it here?

Reply:
Have updated with extra text